haanna/haanna.py

```diff
@@ -229,6 +229,13 @@ def get_heating_status(self, root):
                   + log_type+"']/period/measurement"
         return root.find(locator).text == 'on'
 
+    def get_cooling_status(self, root):
+        """Gets the active cooling status"""
+        if self.is_legacy_anna(root):
+            return False # cooling not supported on legacy Anna
```

Comment:
Better to `return None` to be in line with what HA expects to receive when something does not "exist".